pkg/solidity-utils/contracts/helpers/BalancerErrors.sol

```diff
@@ -134,6 +134,8 @@ library Errors {
     uint256 internal constant AMP_ONGOING_UPDATE = 318;
     uint256 internal constant AMP_FACTOR = 319;
     uint256 internal constant AMP_NO_ONGOING_UPDATE = 320;
+    uint256 internal constant STABLE_INVARIANT_DIDNT_CONVERGE = 321;
+    uint256 internal constant STABLE_GET_BALANCE_DIDNT_CONVERGE = 321;
```

Comment:
These should be added to the .json file. I imagine we have no tests because we don't know how to trigger this condition?

Reply:
Yup, I'm adding all these with along with the amp setting tests